components/Blog/Card/index.tsx

```diff
@@ -0,0 +1,35 @@
+import Description from '@/components/Blog/Card/Description';
+import Img from '@/components/Blog/Card/Img';
+import Info from '@/components/Blog/Card/Info';
+import Title from '@/components/Blog/Card/Title';
+
+interface Props {
+   children: React.ReactNode;
+}
+
+// TODO: Add Card Labels
```

Comment:
we can remomve that, as @oguzergul said, there will be a docs for components soon